apps/website/.vuepress/public/demos/alert/closable-false-ng.html

```diff
@@ -10,9 +10,9 @@
           <clr-icon shape="caret down"></clr-icon>
         </button>
         <clr-dropdown-menu clrPosition="bottom-right">
-          <a href="..." class="dropdown-item" clrDropdownItem>Shutdown</a>
-          <a href="..." class="dropdown-item" clrDropdownItem>Delete</a>
-          <a href="..." class="dropdown-item" clrDropdownItem>Reboot</a>
+          <a href="#" class="dropdown-item" clrDropdownItem>Shutdown</a>
```

Comment:
I'm not sure if this is applicable. But `#` can often cause a page to jump to the top of the screen. I've generally advised people use: `<a href="javascript:void(0)">link-text</a>` instead

Reply:
I've also just omitted the href attribute when I didn't want the anchor to do anything.

Reply:
Yeah I thought about that. I'll replace those.